### ISSUE TITLE
Version built from git gets overwritten with version from AMO

### ISSUE THREAD

**Issue by glitsj16** [uBlockOrigin/uBlock-issues#1484]
### Prerequisites

- [x] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [x] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [x] uBlock Origin is the only extension
    - [x] uBlock Origin with default lists/settings
    - [x] using a new, unmodified browser profile
- [x] I am running the latest version of uBlock Origin
- [x] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description

[Description of the bug or feature]

When building uBlock Origin from git master the current version is `1.33.1.1,` as per https://github.com/gorhill/uBlock/blob/master/dist/version. After installing the newly built xpi in Firefox 85.0 and performing a 'Check for Updates' the browser reports available updates and wants to install uBlock Origin version `1.33.2` from AMO (which is indeed the latest release). IMO the version in https://github.com/gorhill/uBlock/blob/master/dist/version needs to be updated to avoid this.

### A specific URL where the issue occurs

[A specific URL is MANDATORY for issue happening on a web page, even if it happens "everywhere"]

### Steps to Reproduce

1. [First Step]
git clone https://github.com/gorhill/uBlock.git
2. [Second Step]
cd uBlock
sh ./tools/make-firefox.sh
3. [and so on...]
Open about:addons in Firefox.
Click the dropdown on the right and choose 'Install Add-on From File'.
Select the newly built uBlock0@raymondhill.net.xpi.
Shutdown Firefox.
Start Firefox.
Open about:addons in Firefox.
Click the dropdown on the right and choose 'Check for Updates'.

### Expected behavior:

[What you expected to happen]
No available updates for uBlockOrigin.
### Actual behavior:
Firefox wants to update uBlockOrigin as the version on AMO > the version from the git built .xpi.
[What actually happened]

### Your environment

* uBlock Origin version: 1.33.1.1
* Browser Name and version: Firefox 85.0
* Operating System and version: Arch Linux (rolling)

**Comment by gorhill** [comment]
> IMO the version in https://github.com/gorhill/uBlock/blob/master/dist/version needs to be updated to avoid this.

Be detailed, tell me what exactly needs to be updated.

**Comment by glitsj16** [comment]
> Be detailed, tell me what exactly needs to be updated.

https://github.com/gorhill/uBlock/blob/master/dist/version should always be equal to or greater than the version on AMO. Currently that implies changing 1.33.1.1 to 1.33.2 in the version file:
```
--- a/dist/version
+++ b/dist/version
@@ -1 +1 @@
-1.33.1.1
+1.33.2
```

**Comment by glitsj16** [comment]
I've made a PR to fix this: https://github.com/gorhill/uBlock/pull/3783.

**Comment by gorhill** [comment]
How does that solve anything? As soon as I published a version above 1.33.2, Firefox will be back to warn you of an update.

I am declining the issuebhere, just disable auto-update for uBO in Firefox.
